nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/test/java/org/apache/nifi/processors/aws/kinesis/stream/ITConsumeKinesisStream.java

```diff
@@ -102,7 +121,7 @@ private void startKCL(TestRunner runner, InitialPositionInStream initialPosition
     }
 
     @AfterEach
-    public void tearDown() throws InterruptedException {
+    public void tearDown() throws InterruptedException, ExecutionException {
```

Comment:
The `CBOR` env var [has changed](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/migration-client-changes.html) and should be updated in all `ConsumeKinesis` ITs (although I can't recall why this was needed)